drivers/timer/xtensa_sys_timer.c

```diff
@@ -56,13 +56,23 @@ static void ccompare_isr(const void *arg)
 	sys_clock_announce(IS_ENABLED(CONFIG_TICKLESS_KERNEL) ? dticks : 1);
 }
 
+uint32_t xtensa_sys_timer_enabled_mask_3_cnt = 0;
+extern uint32_t my_int_mask_3_cnt;
```

Comment:
Debug code?

Reply:
Yes, I've added a patch for debugging purposes - https://github.com/zephyrproject-rtos/zephyr/pull/35467/commits/ea08a79fb2974dad505e9b7274b95bbd5ca2cd62
I'm debugging the boot failure of SOF with Zephyr and I'm kind of stuck :(
I've explained more here: https://github.com/zephyrproject-rtos/zephyr/pull/35467#issuecomment-860954897